src/sentry/similarity.py

```diff
@@ -143,6 +143,50 @@ def delete(self, scope, items, timestamp=None):
             arguments,
         )
 
+    def export(self, scope, items, timestamp=None):
+        if timestamp is None:
+            timestamp = int(time.time())
+
+        arguments = [
+            'EXPORT',
+            timestamp,
+            len(self.bands),
+            self.interval,
+            self.retention,
+            scope,
+        ]
+
+        for idx, key in items:
+            arguments.extend([idx, key])
```

Comment:
could just do `arguments.extend(itertools.chain(*items))` instead of the for loop, not sure which you prefer; the for loop is certainly more self-explanatory